dashboards/flow-data-for-circuits.json

```diff
@@ -1919,14 +1984,14 @@
           ]
         },
         "datasource": "netsage",
-        "definition": "{\"find\":\"terms\", \"field\":\"meta.sensor_id.keyword\", \"query\":\"-meta.sensor_id:*cenic* AND -meta.sensor_id:*pacificwave*\"}",
+        "definition": "{\"find\":\"terms\", \"field\":\"meta.sensor_id.keyword\", \"query\":\"(meta.flow_type:sflow OR meta.flow_type:netflow OR meta.flow_type:tstat) AND -meta.sensor_id:cenic*\"}",
```

Comment:
Looks like a revert of an earlier commit, maybe this didn't get pulled or merged correctly if conflicts?